rockpaperscissors.c

```diff
@@ -0,0 +1,48 @@
+/* Matthew Danielson
```

Comment:
Your code gave off 9 warnings. You should always try and limit these as much as possible. With that said, your code did run and work properly!